awx/main/management/commands/dump_auth_config.py

```diff
@@ -174,6 +187,22 @@ def handle(self, *args, **options):
                 else:
                     data.append({f"LDAP_{awx_ldap_name}_missing_fields": ldap_missing_fields})
 
+            # dump AZURE settings
+            awx_azure_settings = self.get_awx_azure_settings()
+            awx_azure_enabled, azure_missing_fields = self.is_enabled(awx_azure_settings, self.DAB_AZURE_AUTHENTICATOR_KEYS)
```

Comment:
I see that `is_enabled` is a method that is already established prior to this PR, but I'm concerned that it treats an explicit `None` the same as a missing key/value.  Worse, now that I'm thinking of it, any explicit value that evaluates as false-like.  This is IMO a bug waiting to happen.

Reply:
The way I'd deal with it:

- establish a module variable `MISSING = object()`
- condition becomes `if required and settings.get(key, MISSING) is MISSING:`